### PR TITLE
Allow using raw request body with /code/import.

### PR DESCRIPTION
**Pull request by rwjblue**
As @avdi requested in #2.

### CODE REVIEW

**Comment by joonty** [comment]
Tested, sealed and delivered :)

File: spec/app_spec.rb [hunk 1]
```diff
@@ -31,6 +31,16 @@ def app
 
       expect(last_response).not_to be_ok
     end
+
+    it "can accept codes as raw input" do
+      authorize 'admin', 'ZOMGSECRET'
```
Comment (bf4) [inline spec/app_spec.rb:36]: All your secrets are belong to us

Comment (rwjblue) [inline spec/app_spec.rb:36]: LOL, yeah.  The app will pull from ENV['ADMIN_USERNAME'] and ENV['ADMIN_PASSWORD'] then resort to using 'admin', 'ZOMGSECRET'.

See [here](https://github.com/rubyrogues/ppwm-matcher/blob/master/lib/ppwm_matcher/app.rb#L17)

Comment (bf4) [inline spec/app_spec.rb:36]: You forgot to pass it through the YAML parser.. as JSON embedded in XML... :)

Comment (rwjblue) [inline spec/app_spec.rb:36]: Damn! I knew I was missing something!

**Pull request merged**